lib/client.c

```diff
@@ -514,6 +525,9 @@ _snowflake_check_connection_parameters(SF_CONNECT *sf) {
         log_debug("jwt_timeout: %d", sf->jwt_timeout);
         log_debug("jwt_cnxn_wait_time: %d", sf->jwt_cnxn_wait_time);
     }
+    if (AUTH_OAUTH == auth_type) {
+        log_debug("oauth_token: %s", sf->oauth_token ? "****" : sf->oauth_token);
```

Comment:
in other drivers we want to log only information that sensitive data is provided or not, please change to:
```
log_debug("oauth_token: %s", sf->oauth_token ? "provided" : "not provided");
```

Reply:
Updated.